files/build_templates/docker_image_ctl.j2

```diff
@@ -540,7 +540,9 @@ start() {
     # TODO: Mellanox will remove the --tmpfs exception after SDK socket path changed in new SDK version
 {%- endif %}
     docker create {{docker_image_run_opt}} \
+{%- if '--net' not in docker_image_run_opt %}
```

Comment:
>if [](http://example.com/codeflow?start=4&length=2)

This `if` will possibly override NET related feature in this file. Please unify the solution like docker_image_run_opt and stop using NET.

Reply:
If a docker does not have network attribute, he will have a default value from this file. I used the` if` statement to make code backward compatible.

Reply:
@qiluo-msft, can you please confirm?